server/cri/v1/rpc_container_events.go

```diff
@@ -0,0 +1,9 @@
+package v1
+
+import (
+	pb "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+func (s *service) GetContainerEvents(req *pb.GetEventsRequest, ces pb.RuntimeService_GetContainerEventsServer) error {
```

Comment:
nit: I find it odd we don't have a corresponding comment
`	// Spoofing the GetContainerEvents method to return nil response until actual support code is added.`

Reply:
Done.